dom.bs

```diff
@@ -1405,18 +1407,17 @@ property of the event being dispatched.
    <li><p>Let <var>clearTargetsStruct</var> be the last struct in <var>event</var>'s
    <a for=Event>path</a> whose <a for=Event/path>shadow-adjusted target</a> is non-null.
 
-   <li><p>Let <var>clearTargets</var> be true if <var>clearTargetsStruct</var>'s
-   <a for=Event/path>shadow-adjusted target</a>, <var>clearTargetsStruct</var>'s
-   <a for=Event/path>relatedTarget</a>, or an {{EventTarget}} object in
-   <var>clearTargetsStruct</var>'s <a for=Event/path>touch target list</a> is a <a for=/>node</a>
-   and its <a for=tree>root</a> is a <a for=/>shadow root</a>; otherwise false.
+   <li><p>If <var>clearTargetsStruct</var>'s <a for=Event/path>shadow-adjusted target</a>,
+   <var>clearTargetsStruct</var>'s <a for=Event/path>relatedTarget</a>, or an {{EventTarget}} object
+   in <var>clearTargetsStruct</var>'s <a for=Event/path>touch target list</a> is a <a for=/>node</a>
+   whose <a for=tree>root</a> is a <a for=/>shadow root</a>: set <var>clearTargets</var> to true.
```

Comment:
```suggestion
   whose <a for=tree>root</a> is a <a for=/>shadow root</a>, then set <var>clearTargets</var> to true.
```

Reply:
I did this deliberately as I thought using a colon here would make it clearer. In particular because the clause following "or" is rather long. WDYT?